app/views/full-page-examples/what-is-your-nationality/index.njk

```diff
@@ -47,17 +81,20 @@
               id: "confirm-nationality",
               value: "british-nationality",
               text: "British (including English, Scottish, Welsh or from Northern Ireland)",
-              checked: values["confirm-nationality"] === "british-nationality"
+              checked: britishNationalityValue
             },
             {
               value: "irish-nationality",
               text: "Irish (including from Northern Ireland)",
-              checked: values["confirm-nationality"] === "irish-nationality"
+              checked: irishNationalityValue
```

Comment:
This can be

```js
checked: (
    values["confirm-nationality"] and 
    values["confirm-nationality"].includes("british-nationality")
)
```

Same for the others.